packages/editor-ui/src/components/ResourceMapper/MappingModeSelect.vue

```diff
@@ -0,0 +1,161 @@
+<script setup lang="ts">
+import { ResourceMapperFields, ResourceMapperTypeOptions } from 'n8n-workflow';
+import { computed, getCurrentInstance, ref } from 'vue';
+
+export interface Props {
+	initialValue: string;
+	fieldsToMap: ResourceMapperFields['fields'];
+	inputSize: string;
+	labelSize: string;
+	typeOptions: ResourceMapperTypeOptions | undefined;
+	serviceName: string;
+	loading: boolean;
+	loadingError: boolean;
+}
+
+const instance = getCurrentInstance();
+
+const props = defineProps<Props>();
+
+// Mapping mode options: Labels here use field words defined in parameter type options
+const mappingModeOptions = [
+	{
+		name: instance?.proxy.$locale.baseText('resourceMapper.mappingMode.defineBelow.name'),
```

Comment:
I use locale like this:
1. Import `import { i18n as locale } from '@/plugins/i18n';`
2. Use (both in setup script and template):  `locale.baseText('executionsList.anyStatus')`

Example [here](https://github.com/n8n-io/n8n/blob/environment-git/packages/editor-ui/src/components/ExecutionFilter.vue#L75)